css/style.css

```
@@ -15,5 +14,146 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+/*header*/
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 5rem;
+}
+
+.logo {
+  width: 2rem;
+  height: 2rem;
+}
+/*navbar*/
+nav {
+  display: flex;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  font-size: 20px;
+  margin: 5px;
+  padding: 5px;
+}
+li {
+  align-self: center;
+  padding: 1rem;
+}
+
+a {
+  text-decoration: none;
+  color: grey;
+}
+a:hover {
+  color: orange;
+}
+
+/*main*/
+.main-karma {
+  background-image: url("../img/first-background.jpg");
+  text-align: center;
+  color: white;
+  padding: 15rem;
+}
+button {
+  background-color: orangered;
+  border: none;
+  border-radius: 5px;
+  padding: 7px 15px;
+}
+
+button a {
+  color: white;
+}
 
+.article {
+  text-align: center;
+  margin-top: 30px;
+  font-size: 2rem;
+  font-weight: 400;
+}
+/*picture-img*/
 
+.img-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.refill {
+  width: 135px;
+  height: 145px;
+  text-align: center;
+}
+
+/*get karma*/
+
+.get-karma {
+  display: flex;
+  flex-direction: row;
+  margin: 123px 0px;
+}
+
+/*get-karma-img*/
+.additional {
+  width: 590px;
+  height: 400px;
+}
+
+/*get-karma-rect*/
+.rect-get-karma {
+  background-color: rgba(224, 99, 58, 0.1);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 840px;
+  height: 400px;
+}
+
+/*get-button*/
+.get-today {
+  background-color: orangered;
+  border: none;
+  border-radius: 5px;
+  padding: 7px 15px;
+  justify-content: center;
+  color: white;
+}
+
+/*get-karma-text*/
+.get-karma-text {
+  text-align: center;
+  font-size: 40px;
+  font-style: italic;
+  font-weight: 380;
+  height: 135px;
+  width: 540px;
+}
+
+/*footer*/
+.icon {
+  width: 1rem;
+  height: 1rem;
+  text-align: center;
+  margin: 1rem;
+}
+
+.social-icon {
+  display: flex;
+  justify-content: center;
+}
```

Updated styling for displaying the social icons as in the reference image:

```suggestion
.social-container {
  display: flex;
  justify-content: center;
}

.social-container > a {
    margin: 0 0.5em;
    height: 1em;
    border: solid 1px rgb(212, 212, 212);
    border-radius: 50%;
    padding: 0.5em;
}

.social-container img {
    height: 1em;
}
```

Reference on CSS Border radius: https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius